research/Workshop_1_Write_Up.md

```diff
@@ -10,6 +10,7 @@ Our first workshop brought together 16 researchers (undergraduates, doctoral stu
 ### Time Constants 
 #### Members 
 Chen Li (UK, Masters)  
+Peter Crowe (Germany, Undergraduate)
```

Comment:
I guess I should add the suggestion since it's so simple :smile: 

```suggestion
Peter Crowe (Germany, Undergraduate) 
```

Reply:
ta! need to change a setting there..